pizza_cutter/des_pizza_cutter/_se_image.py

```diff
@@ -384,10 +376,17 @@ def get_psf_image(self, x, y):
         assert np.ndim(x) == 0 and np.ndim(y) == 0, (
             "PSFs are only returned for a single position at a time")
 
-        x = int(x+0.5)
-        y = int(y+0.5)
+        # these are the subpixel offsets of the request position and
+        # the nearest pixel center
+        # we will draw the psf with these same offsets
+        # when used with a coadd via interpolation, this should
+        # locate the PSF center at the proper pixel location in the final
+        # coadd
+        dx = x - int(np.floor(x))
```

Comment:
Should this be `dx = x - int(x+0.5)` ?

Reply:
I'm not saying it should be necessarily, I'm just wondering why you used that formula

Reply:
A different point:  the offsets need to be the same as that used to offset the SE image.  This might be different than the offset of your x,y above, which I think is the center of the patch used for coadding.  In other words, it is how far it gets shifted (at a sub-pixel level).

For example if the center of your patch corresponds to `(row, col) = (351.2, 876.7)` in the SE image, it should be compared to what that will correspond to in the final coadd.